### DISCUSSION TITLE
Controling unsuported camera

### DISCUSSION THREAD

**Discussion by devdems** [ZoneMinder/zoneminder#3789] [Q&A] [answered]
Hi,

i have this camera https://www.amazon.de/-/en/gp/product/B09Y5GC8Q1/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&th=1

It has PTZ functionality and i would like to be able to move the camera in ZoneMinder manually and when the alarm is detected.

Can someone point me in the right direction where to start?

This is how the movement is controlled in the app.
![image](https://github.com/ZoneMinder/zoneminder/assets/8215353/deda3dde-61e0-4b55-a683-00fafbc3ca92)

**Comment by connortechnology** [comment]
It may not be possible, but if it is, you need to figure out how to talk to the camera and send it the commands. Then you can create a control script for it.  We have one for older Wansview cameras, perhaps it may even still work.  https://github.com/ZoneMinder/zoneminder/blob/master/scripts/ZoneMinder/lib/ZoneMinder/Control/Wanscam.pm

**Reply by devdems** [reply]
Hi @connortechnology thx for the reply. Will try this script.

In the meantime i have tried it in Blue Iris and when i added the http link it figured all out i guess from the ONVIF. All the functions that are in the wansview app were also in there. Should ZoneMinder work the same way and i do you think that am just not properly populating the connection attribute?

I did this regarding ONVIF:
![image](https://github.com/ZoneMinder/zoneminder/assets/8215353/66705f5f-d16e-4ad4-a55e-c50e1a8dcb0f)

![image](https://github.com/ZoneMinder/zoneminder/assets/8215353/5ebe996f-078b-434e-bee8-3cf5d9e91608)

**Comment by connortechnology** [comment, marked as answer]
Ifi the camera is ONVIF compliant then yes use either the ONVIF or Netcat control scripts.
